examples/logs/.gitignore

```diff
@@ -0,0 +1,39 @@
+# Local .terraform directories
```

Comment:
Do we need a .gitignore specific to this example?

Reply:
Fixed.